src/__init__.py

```diff
@@ -16,10 +16,15 @@ def merge_configs(a, b):
         if isinstance(a[key], dict) and isinstance(b[key], dict):
             new_config[key] = merge_configs(a[key], b[key])
         else:
-            log.warn(
-                'Overriding keys with different types %s and %s',
-                type(a[key]), type(b[key]))
             new_config[key] = b[key]
+            if a[key] is None:
+                pass
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS108)